### DISCUSSION TITLE
logout-popup-widget not working

### DISCUSSION THREAD

**Discussion by PiyushKrRai** [streetturtle/awesome-wm-widgets#322] [Q&A]
I installed the logout-popup-widget on my system
the widget pops up when i click the specified keybinding but on clicking the buttons on the screen for logout or shutdown nothing happens 
is there something that i'm doing wrong ??
I'm using archlinux btw with awesomewm installed on it

**Comment by streetturtle** [comment]
I've just realized that Discussions doesn't send a notification when it has a new pos :facepalm: I guess I need to configure it properly, sorry for being so late...

If you still interested why it doesn't work (:smiley:) - I would guess the problem is in user permissions - try to run `shutdown now` from terminal under your user. It it requires `sudo` then this is the problem, you'll need to allow your user to run this command. 
Another reason might be that arch uses different command for shutdown and logout, so you'll need to add them as a parameters (`onpoweroff` and `onlogout`)
